clang/test/CodeGenSYCL/offload-fp32-div-sqrt.cpp

```diff
@@ -0,0 +1,229 @@
+// DEFINE: %{common_opts_spirv32} = -internal-isystem %S/Inputs \
+// DEFINE: -fsycl-is-device -emit-llvm -triple spirv32-unknown-unknown
+
+// DEFINE: %{common_opts_spirv64} = -internal-isystem %S/Inputs \
+// DEFINE: -fsycl-is-device -emit-llvm -triple spirv64-unknown-unknown
+
+// DEFINE: %{common_opts_spir} = -internal-isystem %S/Inputs \
+// DEFINE: -fsycl-is-device -emit-llvm -triple spirv-unknown-unknown
+
+// DEFINE: %{common_opts_spir64} = -internal-isystem %S/Inputs \
+// DEFINE: -fsycl-is-device -emit-llvm -triple spirv64-unknown-unknown
```

Comment:
``common_opts_spir64`` seems identical to ``common_opts_spirv64``.

Reply:
Fixed now.